services/movie_session.py

```diff
@@ -42,3 +42,17 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = list(MovieSession.objects.filter(
+        id=movie_session_id
+    ).values("tickets__row", "tickets__seat"))
+    result = []
+    for ticket in tickets:
+        result.append({
+            "row": ticket["tickets__row"],
+            "seat": ticket["tickets__seat"]
+        })
+
+    return result
```

Comment:
You could build a query and avoid iteration by starting from the `Ticket` model, e.g. `Ticket.objects.filter(movie_session_id=...`. Rewrite this method taking this into account